jinseok/routes/postRouter.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const postController = require('../controller/postController');
+
+const router = express.Router();
+
+router.post('/', postController.createPost);
```

Comment:
```suggestion
router.post('', postController.createPost);
```
다른 모든 엔드포인트들도 마찬가지입니다!